src/Component/FieldContainer/FieldContainer.tsx

```diff
@@ -27,65 +27,30 @@
  */
```

Comment:
Did you remove the `TitleField` on purpose? Or is this just an issue with the diff?

Reply:
No, this seems to be a rebasing error. I will have to check this

Reply:
Turns out this was done intentionally. See https://github.com/geostyler/geostyler/pull/1497/commits/ae5240985e1149c3e3d550d81b99aae1ebf82159

Apparently, this component was never used.

Nevertheless, since it was exposed in the `index.ts`, this makes this PR a breaking change. I will update the description accordingly